test/global.test.ts

```diff
@@ -16,7 +17,10 @@ export const longRunningTestsEnabled: boolean = longRunningLocalTestsEnabled ||
 suiteSetup(async function (this: Mocha.Context): Promise<void> {
     this.timeout(120 * 1000);
     await vscode.commands.executeCommand('appService.Refresh'); // activate the extension before tests begin
+
     ext.outputChannel = new TestOutputChannel();
+    registerUIExtensionVariables(ext);
```

Comment:
This should have happened automatically when you activate (refresh) the extension. Why is this being invoked directly here?

Reply:
Refresh wasn't accomplishing that it seems and so was causing an error to be thrown.  I've added an activate block instead which we can use instead of registering these manually.

Reply:
Actually both ways still cause the error to reproduce:
![image](https://github.com/user-attachments/assets/44496613-9fe2-47dd-9384-6a78ed23d885)

Can talk more offline.